code/drasil-printers/Language/Drasil/Printing/Import/Helpers.hs

```diff
@@ -51,15 +53,28 @@ lookupC :: Stage -> ChunkDB -> UID -> Symbol
 lookupC Equational     sm c = eqSymb   $ symbResolve sm c
 lookupC Implementation sm c = codeSymb $ symbResolve sm c
 
--- | Look up a term given a chunk database and a 'UID' associated with the term.
-lookupT :: ChunkDB -> UID -> Sentence
-lookupT sm c = phraseNP $ termResolve sm c ^. term
+-- | Look up a term given a chunk database and a 'UID' associated with the term. Also specifies capitalization
+lookupT :: ChunkDB -> UID -> TermCapitalization -> Sentence
+lookupT sm c NoCap = phraseNP $ termResolve sm c ^. term
```

Comment:
I think it would make sense to create a function from `TermCapitalization` to a `Sentence` transformer, i.e.
```haskell
foo NoCap = phraseNP
foo CapF = atStartNP
foo CapW = titleizeNP
```
(with a better name), and then various other functions here could go back to being 1-liners.  Probably a primed version will be needed too. [Can be done in a new PR]